modules/react/button/lib/PrimaryButton.tsx

```diff
@@ -20,82 +20,100 @@ export interface PrimaryButtonProps extends ButtonProps {
+    },
   },
-  '&:hover, &.hover': {
-    [buttonVars.hover.background]: brand.primary.dark,
-    [buttonVars.hover.border]: 'transparent',
-    [buttonVars.hover.label]: brand.primary.accent,
-    [systemIconStencil.vars.color]: brand.primary.accent,
-  },
-  '&:active, &.active': {
-    [buttonVars.active.background]: brand.primary.darkest,
-    [buttonVars.active.border]: 'transparent',
-    [buttonVars.active.label]: brand.primary.accent,
-    [systemIconStencil.vars.color]: brand.primary.accent,
-  },
-  '&:disabled, &.disabled': {
-    [buttonVars.disabled.background]: brand.primary.base,
-    [buttonVars.disabled.border]: 'transparent',
-    [buttonVars.disabled.label]: brand.primary.accent,
-    [buttonVars.disabled.opacity]: '0.4',
-    [systemIconStencil.vars.color]: brand.primary.accent,
-  },
-});
-
-export const primaryButtonModifiers = createModifiers({
-  variant: {
-    inverse: createStyles({
-      [buttonVars.default.background]: base.frenchVanilla100,
-      [buttonVars.default.borderRadius]: system.shape.round,
-      [buttonVars.default.label]: base.blackPepper400,
-      [systemIconStencil.vars.color]: base.blackPepper400,
-      '&:focus-visible, &.focus': {
-        [buttonVars.focus.background]: base.frenchVanilla100,
-        [buttonVars.focus.label]: base.blackPepper400,
-        [buttonVars.focus.boxShadowInner]: base.blackPepper400,
-        [buttonVars.focus.boxShadowOuter]: base.frenchVanilla100,
-        [systemIconStencil.vars.color]: base.blackPepper400,
-      },
-      '&:hover, &.hover': {
-        [buttonVars.hover.background]: base.soap300,
-        [buttonVars.hover.label]: base.blackPepper500,
-        [systemIconStencil.vars.color]: base.blackPepper500,
-      },
-      '&:active, &.active': {
-        [buttonVars.active.background]: base.soap400,
-        [buttonVars.active.label]: base.blackPepper500,
-        [systemIconStencil.vars.color]: base.blackPepper500,
-      },
-      '&:disabled, &.disabled': {
-        [buttonVars.disabled.background]: base.frenchVanilla100,
-        [buttonVars.disabled.label]: base.blackPepper400,
-        [systemIconStencil.vars.color]: base.blackPepper400,
+  modifiers: {
+    variant: {
+      // Inverse Styles
+      inverse: {
+        [baseButtonStencil.vars.background]: system.color.bg.default,
+        [baseButtonStencil.vars.borderRadius]: system.shape.round,
+        [baseButtonStencil.vars.label]: system.color.text.strong,
+        [systemIconStencil.vars.color]: system.color.fg.strong,
+        // Focus Styles
+        '&:focus-visible, &.focus': {
+          [baseButtonStencil.vars.background]: system.color.bg.default,
+          [baseButtonStencil.vars.label]: system.color.fg.strong,
+          [baseButtonStencil.vars.boxShadowInner]: system.color.fg.strong,
+          [baseButtonStencil.vars.boxShadowOuter]: system.color.fg.inverse,
+          [systemIconStencil.vars.color]: system.color.fg.strong,
```

Comment:
Should icon var use `color.icon` colors?

Reply:
So `fg` is what I was told to use for icons and label inside of buttons